setup.py

```diff
@@ -23,18 +23,21 @@
                sorted(glob.glob(os.path.join('bjoern', '*.c')))
 
 if not WANT_STATSD:
-    SOURCE_FILES.remove('statsd-c-client/statsd-client.c')
-    SOURCE_FILES.remove('bjoern/statsd_tags.c')
+    SOURCE_FILES.remove(os.path.join('statsd-c-client', 'statsd-client.c'))
+    SOURCE_FILES.remove(os.path.join('bjoern', 'statsd_tags.c'))
+
+is_windows = True if os.name == "nt" else False
+
+DEFAULT_COMPILE_ARGS = []
+COMPILE_ARGS = DEFAULT_COMPILE_ARGS if is_windows else DEFAULT_COMPILE_ARGS + [['-std=c99', '-fno-strict-aliasing', '-fcommon', '-fPIC', '-Wall', '-Wextra', '-Wno-unused-parameter', '-Wno-missing-field-initializers', '-g']]
```

Comment:
Are you sure all of these flags must be removed, or maybe a subset suffices?

Reply:
Windows doesn't know any of the flags by default, Or at least for me so I assume these are all Linux flags. IDK if this is maybe the cause of the building wheel error.